xapian-applications/omega/templates/clicklog

```diff
@@ -0,0 +1,3 @@
+$httpheader{Status,302 Found}
+$httpheader{Location,$field{url,$cgi{DOCID}}}
```

Comment:
Similarly, we shouldn't have a blank link between the two `$httpheader` commands here.